src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -138,7 +147,7 @@ public void deleteRoles(final Node node) throws RepositoryException {
         // Read the effective Acl and return a list of acl:Authorization statements
         final List<WebACAuthorization> authorizations = effectiveAcl
                 .map(uncheck(x -> getAuthorizations(x.getLeft().toString())))
-                .orElse(new ArrayList<>());
+                .orElseGet(() -> getDefaultAuthorizations());
```

Comment:
The default `acl:Authorization` will now deny access to all non-admin users, but the wiki should be updated as you describe.